### PR TITLE
Feat/vps breadcrumb

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/solarsteein-dedicated`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

## Description

Add breadcrumb in VPS

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#15257.0 ✘  
* 
  * Build ✘

File: packages/manager/apps/cloud/client/app/app.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ angular
       const dedicatedRedirections = [
         '/paas/veeam-enterprise',
         '/paas/veeam',
-        '/iaas/vps',
+        '/vps',
```
Comment (cbourgois) [inline packages/manager/apps/cloud/client/app/app.js:133]: IMHO, `vps` url in `cloud` always followed `/iaas/vps*`, so we should keep this redirection pattern as this (if someone had a bookmark on it for example, but nobody have `/vps` bookmark on `cloud`). And we should redirect from `/iaas/vps` to `/vps` in `dedicated` for same reasons.


Redirection should works like this : `cloud/#/iaas/vps` => `dedicated/#/iaas/vps` => `dedicated/#/vps`

What do you think ?

**Review by cbourgois — APPROVED** [review]

File: packages/manager/apps/cloud/client/app/app.js (hunk 1 above)
Comment (marie-j) [reply packages/manager/apps/cloud/client/app/app.js:133]: @cbourgois I agree, I did not pay attention while doing a batch replace but you're right, I'll update

**Comment by marie-j** [comment]
Rebase on `feat/solarsteein`

**Review by antleblanc — APPROVED** [review]

**Pull request merged**